### ISSUE TITLE
Dual-dating dates are converted incorrectly

### ISSUE THREAD

**Issue by dthaler** [gedcom7code/c-converter#5]
The date conversion is outputting the Julian year as if it were the Gregorian year.

Input:
```
2 DATE 3 MAR 1751/2
```

Expected output:
```
2 DATE 3 MAR 1752
3 PHRASE 3 MAR 1751/2
```

Actual output:
```
2 DATE 3 MAR 1751
3 PHRASE 3 MAR 1751/2
```

**Comment by tychonievich** [comment]
I've queued this for fixing, but it will require a minor refactoring of geddate.c; in particular, I'll need to change how I handle slashes depending on where they appear.

**Comment by tychonievich** [comment]
I have a possible fix in PR #8 ; just need to run a few more tests before committing it to the main branch
